packages/itwinui-react/src/core/FileUpload/FileUploadCardAction.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import cx from 'classnames';
+
+export type FileUploadCardActionProps = {
+  /**
+   * Buttons in the dialog bar.
+   */
+  children: React.ReactNode;
```

Comment:
many of these comments are incorrectly copied, but also, you don't need to explicitly add `children` (in any of the files, not just this one). It will automatically come from `React.ComponentProps`

Reply:
also it might be easier to navigate if you move all subcomponents in the same file, similar to [`Flex`](https://github.com/iTwin/iTwinUI/blob/main/packages/itwinui-react/src/core/utils/components/Flex.tsx)

Reply:
The comments and the explicitly adding of `children` have been removed. All of the sub-components have been moved to the same file